app/communities-configs/filecoin/index.js

```diff
@@ -0,0 +1,318 @@
+        :first-child {
+          margin-left: 0;
+        }
+
+        :last-child {
+          margin-left: auto!important;
+          padding: 0!important;
+        }
+      }
+
+      margin: 0 !important;
+    `,
+    CustomSubHeader: `
+      margin: 0 !important;
+    `,
+    subHeaderContainerStyles: `
+      // margin: 0 !important;
+    `,
+    topLogoContainerStyles: `
+      width: 80px!important;
+      margin: 0!important;
+      margin-right: 60px!important;
+      display: flex;
+      justify-content: center;
+      align-content: center;
+    `,
+    subHeaderLogoStyles: `
+      height: 40px;
+
+      > img {
+        height: 100%;
+      }
+    `,
+
+    Highlighted: `
+      color: #000000 !important;
+      border: solid 1px rgba(0,0,0,.15);
+      border-radius: 15px;
+      padding: 5px 15px !important;
+    `,
+  },
+  links: [
+    {
+      text: 'Store',
+      href: `${domainName}/store/`,
+    },
+    {
+      text: 'Provide',
+      href: `${domainName}/provide/`,
+    },
+    {
+      text: 'Build',
+      href: `${domainName}/build/`,
+      subitems: [
+        {
+          text: "Docs",
+          href: `${domainName}/build/#benefits-banner`
+        },
+        {
+          text: "Tools & services",
+          href: `${domainName}/build/#tools-and-more`
+        },
+        {
+          text: "Grants",
+          href: `${domainName}/build/#grants`
+        },
+        {
+          text: "Roadmap",
+          href: `${domainName}/build/#usp`
+        },
+        {
+          text: "Videos",
+          href: `${domainName}/build/#videos`
+        },
+        {
+          text: "Filecoin Community",
+          href: `${domainName}/build/#community`
+        },
+        {
+          text: "Events",
+          href: `${domainName}/build/#events`
+        },
+      ]
+    },
+    {
+      text: 'Blog',
+      href: `${domainName}/provide/`,
+    },
+    {
+      text: 'Explore the Network',
+      isHighlighted: true,
+      subitems: [
+        {
+          text: 'Filfox.info',
+          href: 'https://filfox.info/en',
+        },
+        {
+          text: 'Filscan.io',
+          href: 'https://filscan.io/#/tipset/chain',
+        },
+        {
+          text: 'Filscout.io',
+          href: 'https://filscout.io/en/',
+        },
+        {
+          text: 'Spacegap',
+          href: 'https://spacegap.github.io/',
+        },
+      ],
+    },
+  ],
+};
+
+export const FilecoinStyles = {
+  withoutCopyright: true,
+  withoutSubHeader: true,
+  poweredByPeeranha: true,
+  withoutAdditionalLinks: true,
+  signUpPageLogo: FileiconLogo,
+  favicon: favicon,
+  mobileSubHeader: (
+    <CustomMobileSubHeader config={CustomSubHeaderConfig} logo={FileiconLogo} />
+  ),
+  customSubHeader: <CustomSubHeader config={CustomSubHeaderConfig} />,
+  withoutFAQ: true,
+  fonts: {
+    h3LetterSpacing: '1px',
+    tagsLetterSpacing: '1px',
+    questionTitleLetterSpacing: '1px',
+    h3: 'Roboto-Regular, Arial, sans-serif',
```

Comment:
`Roboto-Regular, Arial, sans-serif` repeats for than 5 times for this file, extract to variable